bin/terminus.php

```diff
@@ -28,8 +28,7 @@
 $container = new Container();
 $input = new ArgvInput($_SERVER['argv']);
 $output = new ConsoleOutput();
-$roboConfig = new \Robo\Config(); // TODO: make Terminus Config extend \Robo\Config and use $config here
-Robo::configureContainer($container, $roboConfig, $input, $output, $application);
+Robo::configureContainer($container, $config, $input, $output, $application);
```

Comment:
No, his base branch is OOD.
https://github.com/pantheon-systems/terminus/blob/master/bin/terminus.php